helpers/config.py

```diff
@@ -780,6 +793,50 @@ def __questions_aws(self):
             self.__dict['aws_secret_key'] = ''
             self.__dict['aws_bucket_name'] = ''
 
+    def __questions_aws_validate_credentials(self):
+        """
+        Prompting user whether they would like to validate their entered AWS
+        credentials or continue without validation.
+        """
+        aws_credential_attempts = 0
+
+        if self.__dict['use_aws']:
+            self.__dict['aws_validate_credentials'] = CLI.yes_no_question(
+                'Would you like to validate your AWS credentials?',
+                default=self.__dict['aws_validate_credentials'],
+            )
+
+        if self.__dict['use_aws'] and self.__dict['aws_validate_credentials']:
+            while not self.__dict['aws_credentials_valid'] and (
+                aws_credential_attempts <= self.MAXIMUM_AWS_CREDENTIAL_ATTEMPTS
+            ):
+                aws_credential_attempts += 1
+                self.validate_aws_credentials()
+                attempts_remaining = self.MAXIMUM_AWS_CREDENTIAL_ATTEMPTS - \
```

Comment:
We usually to keep the operator on the second line 
```python3
                attempts_remaining = (self.MAXIMUM_AWS_CREDENTIAL_ATTEMPTS
                                      - aws_credential_attempts)
```

Reply:
Going with the Black way, this is what I went with instead.

```python
attempts_remaining = (
    self.MAXIMUM_AWS_CREDENTIAL_ATTEMPTS
    - aws_credential_attempts
)
```